StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -281,7 +282,7 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
                         error: FinancialConnectionsSheetError.unknown(debugDescription: "STPAPIClient deallocated.")
                     )
                 }
-                return self.post(resource: APIEndpointAuthSessionsAccounts, object: body)
+                return self.post(resource: APIEndpointAuthSessionsAccounts, parameters: body)
```

Comment:
what's this change? object is of different type?

Reply:
The addition of the array changed body to be `[String: Any]`.

`object` expects Encodable/Decodable, which `Any` is not

`parameters` accepts `[String:Any]`